src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravRepository.kt

```diff
@@ -34,6 +37,29 @@ class AktivitetskravRepository(private val database: DatabaseInterface) {
             }
         }
 
+    fun createAktivitetskrav(
+        newAktivitetskrav: Aktivitetskrav,
+        previousAktivitetskrav: UUID?,
+    ): PAktivitetskrav {
+        val createdRecord = database.connection.use { connection ->
+            connection.prepareStatement(CREATE_AKTIVITETSKRAV_NY_VURDERING).use {
+                it.setString(1, newAktivitetskrav.uuid.toString())
+                it.setObject(2, newAktivitetskrav.createdAt)
+                it.setObject(3, newAktivitetskrav.createdAt)
+                it.setString(4, newAktivitetskrav.personIdent.value)
+                it.setString(5, newAktivitetskrav.status.name)
+                it.setDate(6, Date.valueOf(newAktivitetskrav.stoppunktAt))
+                it.setString(7, null)
+                it.setString(8, previousAktivitetskrav.toString())
```

Comment:
Tror man bør gjøre `?.toString` her ellers lagres strengen `null` som potensielt kan bli problemer... kan evt sjekke det i en test.

Reply:
Lager en test for å sjekke 😊